custom_components/noonlight/__init__.py

```diff
@@ -29,6 +29,17 @@
 CONF_SECRET = 'secret'
 CONF_API_ENDPOINT = 'api_endpoint'
 CONF_TOKEN_ENDPOINT = 'token_endpoint'
+CONF_LINE1 = 'line1'
+CONF_LINE2 = 'line2'
+CONF_CITY = 'city'
+CONF_STATE = 'state'
+CONF_ZIP = 'zip'
+CONF_NAME = 'name'
+CONF_PHONE = 'phone'
+CONF_PIN = 'pin'
```

Comment:
`CONF_PIN` can be imported from https://github.com/home-assistant/core/blob/dev/homeassistant/const.py#L138

Reply:
update included in my copy